src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -234,66 +235,107 @@ export class StackDetailsComponent implements OnInit {
     this.currentStackRows = [];
     for (let component in components) {
       if (components.hasOwnProperty(component)) {
-        this.currentStackRows.push({ name: components[component].name, version: components[component].version });
+        this.currentStackRows.push({ name: components[component].name,
+          version: components[component].version });
       }
     }
   }
 
+  // TODO: To be removed after the demo
+  private fetchStaticRecommendation(): any {
+    return {
+            'recommendations': {
+                'similar_stacks': [
+                    {
+                        'analysis': {
+                            'version_mismatch': {
+                                'vertx:vertx-web-templ-freemarker': '3.3.4',
+                                'vertx:vertx-web-templ-mvel': '3.4.0'
+                            },
+                            'missing_packages': {
+                                'vertx:vertx-mongo-embedded-db' :'3.3.3'
+                            }
+                        },
+                        'similarity': 0.7009090909090909,
+                        'uri': 'http://cucos-01.lab.eng.brq.redhat.com:32100/api/v1.0/appstack/18'
+                    }
+                ]
+            }
+        };
+  }
 
+  private setComponentsToGrid(stackData: any): void {
+    let components: Array<any> = stackData.components;
+    let length: number = components.length;
+    for (let i = 0; i < length; i++) {
+      let myObj: any = {};
+      myObj.ecosystem = components[i].ecosystem;
```

Comment:
@nimishamukherjee The names are different in both the cases and hence the change.